### ISSUE TITLE
Stop Words suggestion

### ISSUE THREAD

**Issue by DruidSmith** [davidmogar/normalizr#5]
There isn't much documentation on how to use the stop-words list - and would it make sense to add the capability to use a custom stop-word list rather than having to modify an existing one?  Or does that capability already exist?

**Comment by davidmogar** [comment]
You are more than right about the custom lists. At the moment there is nothing like that bu could be added easily. I'll find some time to do it. Thanks for your suggestion ;)

**Comment by davidmogar** [comment]
I should make this easier, but you could find the path to stop-words files and create a file named stop-custom. After that you should only set the language to `custom` when initialising Normalizr:

```python
from normalizr import Normalizr

normalizr = Normalizr(language='custom')
```

I'm leaving this issue open till I decide what to do ;)

**Comment by DruidSmith** [comment]
Will give it a try....

**Comment by JasonCrowe** [comment]
Adding my 2 cents...

I don't have a use for my own custom stopword list, but it would be nice to be able to add words to the stop list with the normalization settings. IE.

    normalizations = [
        'remove_extra_whitespaces',
        'remove_stop_words',
        ('add_stopwords', [ 'stopword_1',  'stopword_2',  'stopword_3',  'stopword_4', ])
    ]

**Comment by davidmogar** [comment]
Makes sense. I'll think about it and come up with something. I still have to find some time to implement changes in cucco (really needed ones).

EDIT: Thank you btw!
